lib/cluster.js

```diff
@@ -186,7 +191,11 @@ Cluster.prototype.fetchStats = function fetchStats(callback) {
             self.parseStats(stats);
         });
 
+        if (self.connectionErrorCount > 0) {
+            console.error('** Failed to connect to %d nodes.\n', self.connectionErrorCount);
```

Comment:
I don't think this is great. If we're going to improve error handling, we should do better than just a line on top. Every command should be able to treat errors according to the output/functionality it's providing.